packages/contracts/docs/osx/index.md

```diff
@@ -4,9 +4,9 @@ title: aragonOSx
 
 ## The Contracts behind aragonOSx
```

Comment:
```suggestion
## The Contracts behind Aragon OSx
```

Reply:
Fixed in 66151f518d01899579276fd6df43401e0a0e1d79